app/models/recipe_product.rb

```diff
@@ -1,4 +1,5 @@
 class RecipeProduct < ApplicationRecord
   belongs_to :recipe
   belongs_to :product
+  counter_culture :recipe, column_name: 'calories', delta_column: 'calories'
```

Comment:
I don't really see the benefits of using this gem, but maybe we can discuss this during the call

Reply:
Calories calculator works for recipe_products - multiply quantity and calories and save the calories of each recipe_product. counter_culture sum calories of all recipe_products connected with selected recipe.